dae-alright/src/App.js

```diff
@@ -1,21 +1,15 @@
-import React, { Component } from 'react';
 
+import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import './App.css';
+import HeaderWraper from "../src/components/Header/HeaderWraper/index"
```

Comment:
You can remove the index from this import because `js` will automatically import it

Reply:
OKay. Thanks

Reply:
Done